api/v1alpha1/sysbench_types.go

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright 2019 The xridge kubestone contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// SysbenchSpec defines the desired state of Sysbench
+type SysbenchSpec struct {
+	// Image defines the sysbench docker image used for the benchmark
+	Image ImageSpec `json:"image"`
+
+	PodConfigurationSpec `json:",inline"`
+
+	// Options is a list of zero or more command line options starting with '--'.
```

Comment:
Initially looking at this I thought you need to something like this:
`-- --param1=blah`, which isn't the case. 

Also, I'd opt for uniformity here: two other benchmarks are using `cmdLineArgs`, this one goes with `options`.
I don't mind which one is taken, but I'm leaning towards to `...Args` (... can be different from cmdLine) as args encompasses options and parameters: https://stackoverflow.com/questions/36495669/difference-between-terms-option-argument-and-parameter

Meta point is: it should be consistent across all the three benchmarks.

Reply:
Sorry, I see why we are going with Options, TestName and Command:
```
sysbench [options]... [testname] [command]
```

Reply:
Also seeing that the description was taken from sysbench. I'd suggest the sysbench calling convention to the documentation so that confusion can be avoided.

Reply:
Got it. I will extend the documentation with it.